pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/FunctionStats.java

```diff
@@ -18,149 +18,91 @@
  */
 package org.apache.pulsar.functions.instance;
 
+import com.google.common.collect.EvictingQueue;
+import io.prometheus.client.Counter;
+import io.prometheus.client.Summary;
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.pulsar.functions.proto.InstanceCommunication;
 
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-
 /**
  * Function stats.
  */
 @Slf4j
 @Getter
 @Setter
 public class FunctionStats {
+
+    private static final String[] metricsLabelNames = {"tenant", "namespace", "name", "instance_id"};
+
+    /** Declare Prometheus stats **/
+
+    static final Counter statTotalProcessed = Counter.build()
+            .name("__function_total_processed__")
```

Comment:
my personal preference would be to keep the names with similar conventions we are already using for all the other components (and what others are dumping into prometheus as well). 

Like `__function_total_processed__` --> `pulsar_function_total_processed` 
... 

It's easy to avoid conflict by applying label on the user generated metrics in any case.

Reply:
@srkukarni what do you think